bson/__init__.py

```diff
@@ -113,13 +120,10 @@
 from bson.regex import Regex
 from bson.son import RE_TYPE, SON
 from bson.timestamp import Timestamp
-from bson.tz_util import utc
```

Comment:
We need to keep this import to maintain the `from bson import utc` api. To avoid a "imported but not used" error I think we can do:
```
from bson.tz_util import utc as utc
```